extensions/rename.md

```diff
@@ -0,0 +1,128 @@
+---
+title: IRCv3 `rename` Extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Sadie Powell"
+    email: "sadie@witchery.services"
+    period: "2017-2018"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the unprefixed `rename` capability name. Instead, implementations SHOULD use the `draft/rename` capability name to be interoperable with other software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Introduction
+
+Historically there has been no reasonable way to rename a channel without closing down the channel and redirecting to a new one. This isn't ideal since properties such as access and ban lists need to be manually copied to the new channel. This also causes problems for client-side logging where history is split across multiple channel names.
+
+This specification introduces a method of natively renaming channels that does not suffer from these drawbacks.
+
+## Architecture
+
+### Capabilities
+
+This specification introduces the `draft/rename` capability. This capability informs the server that the client is capable of handling `RENAME` messages.
+
+### Messages
+
+This specification adds the `RENAME` command. This command allows a client to instruct the server to change the name of a channel, and in turn allows a server to inform a client that a channel name has been changed.
+
+A channel rename preserves all channel state, such as membership, mode and topic.
+
+The `RENAME` command has between two and three parameters. The first parameter is the current channel name, the second parameter is the new channel name, and the optional third parameter is a reason for renaming the channel.
+
+Server implementations SHOULD NOT allow channels to be converted between types with the `RENAME` message. Client implementations SHOULD be able to handle renaming between channel types.
+
+Implementations MUST allow channels to be renamed while only changing the casing of a channel name.
+
+### Errors
+
+This specification defines `FAIL` messages using the [standard replies][] framework for notifying clients of errors with channel renaming. The following codes are defined:
+
+* `FAIL RENAME CHANNEL_NAME_IN_USE <old-channel> <new-channel>: The channel name is already taken`
```

Comment:
If `The channel name is already taken` is part of the predefined text, move `:` to the right. If it's not, move the `` ` `` to left of `:`.